src/includes/classes/AutoCache.php

```diff
@@ -282,6 +285,11 @@ protected function getSitemapUrlsDeep($sitemap, $___recursive = false)
             }
             unset($_sitemapindex_urls, $_urlset_urls);
         }
+
+        if (!$allow_url_fopen) {
```

Comment:
Yes, I updated that in https://github.com/websharks/comet-cache-pro/pull/264/commits/6219b3515a37f7f214b25d86cec918b8f9954386; so you're looking at an old version. :-)